src/tests/controllers/api/subclassController.test.js

```diff
@@ -101,19 +101,19 @@ describe('show', () => {
 describe('showLevelsForSubclass', () => {
   const findDoc = [
     {
-      index: 1,
```

Comment:
Actually, do you mind just switching this to `berserker` while you're here? I think it's reasonable to adjust it in this case since it's only test text.

Reply:
Yep, not a problem!